src/topics/lda_scvb.cpp

```diff
@@ -184,5 +185,18 @@ double lda_scvb::compute_doc_topic_probability(learn::instance_id doc,
     return (doc_topic_count_.at(doc).at(topic) + alpha_)
            / (doc_sizes_.at(doc) + num_topics_ * alpha_);
 }
+
+stats::multinomial<topic_id> lda_scvb::topic_distrbution(doc_id doc) const
+{
+    // TODO: Replace the count vectors with a multinomial rather than creating
+    // it here
+    stats::multinomial<topic_id> result = stats::multinomial<topic_id>();
+    for (topic_id tid{0}; tid < num_topics_; ++tid)
+    {
+        result.increment(tid, doc_topic_count_.at(doc).at(tid));
```

Comment:
It probably only matters if you intend on using `dist.counts(event)` later on and expect it to give you an integer. The probability values coming out of `dist.probability(event)` would be equivalent (it just returns `counts(event) / counts()` which would end up being `probability(event) / 1.0` if you use `increment()` with probabilities instead of counts.

The alternative would just be

```cpp
result.increment(tid, doc_topic_count_.at(doc).at(tid) + alpha_);
```